Breast Cancer Prediction

murtaza-sadri-19 opened the pull request:
#33
### Checklist
<!-- You can add more relevant points -->
- [x] My code follows the current [project structure](https://github.com/yashasvini121/predictive-calc/blob/master/docs/project-structure.md)
- [x] I have thoroughly reviewed and updated the `requirements.txt` file to include any new packages
- [x] The `predict.py` file includes a properly implemented `model_details()` function, or the notebook contains this function to print a detailed model report. **The model will not be accepted without this function**, as it is essential for generating the necessary model details.
- [x] I have added relevant tests (if necessary).
- [x] I have added comments in the code where needed.
- [x] This PR is submitted under **Hacktoberfest**.
- [x] This PR is submitted under **GirlScript Summer of Code (GSSoC-Extd)**.

murtaza-sadri-19 commented:
@yashasvini121 Please review this

yashasvini121 commented on models/Breast_Cancer_Detector/predict.py:
```diff
@@ -0,0 +1,102 @@
+
+    st.write("Predicted cancer type:", prediction)
+
+def model_details():
+    st.write("## Breast Cancer Detector Model Details")
+    st.write("### Model Type")
+    st.write("The Breast Cancer Detector model is a machine learning model that uses a classification algorithm to predict whether a tumor is malignant or benign based on a set of input features.")
+    st.write("### Features")
+    st.write("The model uses the following features to make predictions:")
+    st.write("* Mean radius")
+    st.write("* Mean texture")
+    st.write("* Mean perimeter")
+    st.write("* Mean area")
```
Remove this fxn.

yashasvini121 commented on models/Breast_Cancer_Detector/predict.py:
```diff
@@ -0,0 +1,102 @@
+from model import Loaded_model, cancer_type_prediction
```
Update the file paths:
`from models.Breast_Cancer_Detector.model import Loaded_model, cancer_type_prediction
`

yashasvini121 reviewed (changes requested):
You have not made any changes to your code since the last review. Let me know if you face any errors

murtaza-sadri-19 commented:
> You have not made any changes to your code since the last review. Let me know if you face any errors
I have updated the model path and details in pages.json and removed model_details.

I have updated model path and model details in pages.json
I have removed model_details function

Please check commit [5b9abae2db94420edaae4089cd4c803650a0cad7](https://github.com/your-repo/your-project/commit/5b9abae2db94420edaae4089cd4c803650a0cad7) for the changes. Let me know if you encounter any errors!

murtaza-sadri-19 commented:
> You have not made any changes to your code since the last review. Let me know if you face any errors

I not getting how to add model details into notebook as you said to do.
